source/grid/tria_accessor.cc

```diff
@@ -1082,6 +1082,102 @@ set (const internal::Triangulation::TriaObject<structdim> &object) const
 
 
 
+template <>
+TriaAccessor<0,1,1>
+TriaAccessor<1,1,1>::vertex_accessor (const unsigned int i) const
```

Comment:
Is there no way to unify at least some of these functions? Why is it necessary to completely specialize them all?